### DISCUSSION TITLE
How do i add recommended blog posts in existing article

### DISCUSSION THREAD

**Discussion by newbeelearn** [nunocoracao/blowfish#481] [Q&A] [answered]
I want to recommend additional posts that may be relevant to the reader when they read the article recommendations can be based on likes/views or similar tags. I can only see next and previous. I tried looking for parameters but could not figur it out. Is this supported in this theme?

**Comment by nunocoracao** [comment, marked as answer]
Hello @newbeelearn, this feature is not supported at the moment. And it will most likely will not ever be. Hugo is meant to generate a static website, this kind of personalization requires a different stack.

**Reply by newbeelearn** [reply]
Ok. Static websites can have recommendations based on similar keyword or similar tags both of which are supported in this theme so my guess is it should be possible but i am not sure. Personalization for individual reader is not possible because sites are generated statically but recommendation based on content should be possible what do you think?

**Reply by nunocoracao** [reply]
Got you. There is a List partial which can be used already. Potentially we can extend that partial to support additional use-cases, feel free for to open a ticket with your request
